equippedgear/src/ts/components/equippedgear-window.tsx

```diff
@@ -0,0 +1,176 @@
+  }
+
+  unequipItem(item: Item): void {
+    client.UnequipItem(item.id);
+  }
+
+  render() {
+    const items: any[] = [];
+
+    this.gearSlots.forEach((slotId: string, index: number) => {
+      const item = this.state.equippedgear.getItemInGearSlot(slotId);
+      if (item != null) {
+        items.push((
+          <li key={'gear-slot'+index} className="gear-slot-title cu-font-cinzel">{this.getGearSlotName(slotId)}</li>
+        ));
+        items.push((
+          <Tooltip placement="topLeft" key={'item-tooltip' + index} overlay={this.renderTooltip.call(this, item)} arrowContent={<div className="cu-tooltip-arrow-inner"></div>} prefixCls="cu-tooltip" mouseLeaveDelay={0} mouseEnterDelay={0.25}>
+            <li key={'item'+index} onDoubleClick={this.unequipItem.bind(this, item)} onContextMenu={this.unequipItem.bind(this, item)}>
+              <div className="icon"><img src="../../interface-lib/camelot-unchained/images/items/icon.png" /></div>
+              <div className="name">{item.name}</div>
+            </li>
+          </Tooltip>
+        ))
+      }
+    });
+    return (
+      <div className="cu-window">
+        <div className="cu-window-header">
+          <div className="cu-window-title">Equipped Gear</div>
+          <div className="cu-window-actions">
+            <a onMouseDown={this.closeWindow.bind(this)} className="cu-window-close"></a>
+          </div>
+        </div>
+        <div className="cu-window-content">
+          <ul className="equippedgear-list list-vertical">
+            {items}
+          </ul>
+        </div>
+      </div>
+    );
+  }
+
+  renderTooltip(item: Item) {
+    return (
+      <table className="cu-table tooltip-content">
+        <tbody>
+          <tr>
+            <th>Name</th>
+            <td>{item.name}</td>
+          </tr>
+          <tr>
+            <th>Description</th>
+            <td>{item.description}</td>
+          </tr>
+          <tr>
+            <th>Type</th>
+            <td>{this.getItemTypeName(item.type)} ({item.type})</td>
+          </tr>
+          <tr>
+            <th>Gear Slot</th>
+            <td>{this.getGearSlotName(item.gearSlot)} ({item.gearSlot})</td>
+          </tr>
+          <tr>
+            <th>Item ID</th>
+            <td className="font-monospace font-small">{item.id}</td>
+          </tr>
+          <tr>
+            <th>Resource ID</th>
+            <td className="font-monospace">{item.resourceID}</td>
+          </tr>
+          <tr>
+            <th>CarryRequirement</th>
+            <td>{item.carryingRequirement}</td>
+          </tr>
+        </tbody>
+      </table>
+    );
+  }
+
+  getItemTypeName(type: itemType): string {
+    switch (type) {
+      case itemType.NONE:
+        return 'None';
+      case itemType.EQUIPABLE:
+        return 'Equipable';
+      case itemType.RESOURCE:
+        return 'Resource';
+      default:
+        return 'None';
+    }
+  }
+
+  getGearSlotName(slot: any): string {
+    switch (parseInt(slot, 10)) {
```

Comment:
Not yet.  We have chosen the library.  This is something I could pick up and do.  I think doing it sooner rather than later would be a good idea.

https://trello.com/c/c2nd6BRK/79-camelot-unchained-lib-i18n